src/Compilers/Core/Portable/Symbols/Attributes/AttributeUsageInfo.cs

```diff
@@ -167,11 +169,11 @@ internal object GetValidTargetsErrorArgument()
 
         private struct ValidTargetsStringLocalizableErrorArgument : IFormattable
         {
-            private readonly string[] _targetResourceIds;
+            private readonly string[]? _targetResourceIds;
```

Comment:
I'm not understanding why this is marked nullable? We're asserting that it's never assigned null in the constructor. We have an extraneous null check in the `ToString` method, yes, but that's not a reason to mark this nullable when it clearly is not.

Reply:
I am also curious what warnings come up when this annotation is removed

Reply:
`new ValidTargetsStringLocalizableErrorArgument()` produces an instance with this set to null, and the type is implemented such that the case is handled without failures. The field was updated to match.